website/src/Home.js

```diff
@@ -1,26 +1,21 @@
 import m from 'mithril'
-import { Menu, Links } from './Menu'
 
-function Home() {
-  return {
-    view: (vnode) => {
-      return [
-        m(Menu),
-        m('span#main', {}, [
-          m('h3.title', {align: 'center'}, 'Alfred: the automatic video editor'),
-        ]),
-        m(Links),
```

Comment:
I removed the `Menu` and `Links` because those are included in the `Layout` component. This was duplicating the menu on the home page.